docs/ai/how-to/app-service-aoai-auth.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Authenticate and Authorize App Service to Azure OpenAI using Microsoft Entra and the Semantic Kernel SDK"
+description: "Learn how to authenticate and authorize your app service application to an Azure OpenAI resource using Microsoft Entra managed identities and the Semantic Kernel SDK for .NET."
+author: haywoodsloan
+ms.topic: how-to
+ms.date: 04/19/2024
+zone_pivot_groups: azure-development-tool-set-two
+
+#customer intent: As a .NET developer, I want authenticate and authorize my App Service to Azure OpenAI using Microsoft Entra so that I can securely use AI in my .NET application.
+
+---
+
+# Authenticate and authorize App Service to Azure OpenAI using Microsoft Entra and the Semantic Kernel SDK
+
+This article demonstrates how to use [Microsoft Entra managed identities](/azure/app-service/overview-managed-identity) to authenticate and authorize an App Service application to an Azure OpenAI resource.
+
+By using a managed identity from Microsoft Entra, your App Service application can easily access protected Azure OpenAI resources without having to manually provision or rotate any secrets.
+
+The [Semantic Kernel SDK](/semantic-kernel/overview) enables you to easily implement Microsoft Entra authentication in your .NET application.
+
+## Prerequisites
+
+* An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* [.NET SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
+* [`Microsoft.SemanticKernel` NuGet package](https://www.nuget.org/packages/Microsoft.SemanticKernel)
+* [`Azure.Identity` NuGet package](https://www.nuget.org/packages/Azure.Identity)
+* [Create and deploy an Azure OpenAI Service resource](/azure/ai-services/openai/how-to/create-resource)
+* [Create and deploy a .NET application to App Service](/azure/app-service/quickstart-dotnetcore)
+
+## Add a managed identity to App Service
```

Comment:
Empty headers are usually discouraged - can we add a couple sentences in here reminding users what system-assigned vs user-assigned identities are?

Reply:
Added a description of system vs user assigned identities to both docs.